NewEvaluation.html

```diff
@@ -654,20 +643,18 @@
 			</fieldset>
 		</form>
 
-	<div class="littleSpacing"></div>
-
-	<form  action="Evaluation.html">
-		<input  type="submit" name="button" value="Submit">
-	</form>
+		<div class= "submit-button">
+			<form action="Evaluation.html">
+				<input  type="submit" name="button" value="Submit">
```

Comment:
So i should find a way to hook all the data and aend it in a specific way to the server ?